### ISSUE TITLE
Missing txs from account in explorer

### ISSUE THREAD

**Issue by sanchopansa** [MinaProtocol/mina#6787]
As part of Testnet 4.1, I broadcast 2 txs to the echo service.

**Steps to reproduce**

Unlocking account:
```
vganev_factomatic_io@mina:~$ coda accounts unlock -public-key $CODA_PUBLIC_KEY
Password to unlock account: 

🔓 Unlocked account!
Public key: B62qoeGtw6SzaRahbprfy1RUgQBGHv7cRbzpPn5tLXdTxLSsL5R5q5o
```

First tx:
```
vganev_factomatic_io@mina:~$ coda client send-payment \
>   -amount 0.5 \
>   -receiver B62qndJi5mnRoBZ8SAYDM1oR2SgAk5WpZC8hGpJUZ4e64kDHGbFMeLJ \
>   -fee 0.1 \
>   -sender $CODA_PUBLIC_KEY
Dispatched payment with ID G8k9HJXpug4J4thR9me9Ft7CfDwWuBX4CEr49173KRybGMdzqijSfkvRHHWWy4scNMLVm29ytvPhu6ZD6zFeKBfPJMt52eWvFWFGEAru6AiWQmTM7ScwzwcHb3UeEE8Xe3WaqMwVo8G3uWkduHbyhErekYbhUV6BeVDRUH1tgTYxBJDj8HnShwLASXKumVNj2HrQu8Lc6QNpXmYMKHu8m89g2HaVeu43D2u6KeV5h2wGAhycVmS82ikEqSHUrVXVrwsqfLeY9R2crB8o2DddPyMRexZsQAMxWo4ko2xbno2xe3fE53BiRZGARKpQBTKz4FzP6JrjiDmcbjVpfdfNEvJ1ngmp5FGRHESBRtKrpXpz3UezAUdguQH2tnjFHwXppP7fxSDaY
```

Second tx:
```
ganev_factomatic_io@mina:~$ coda client send-payment -amount 42.69 -receiver B62qndJi5mnRoBZ8SAYDM1oR2SgAk5WpZC8hGpJUZ4e64kDHGbFMeLJ -fee 0.1 -sender $CODA_PUBLIC_KEY
Dispatched payment with ID 2i2eUzkw33QCS2uBKkT4Xypvf25vsmtZBZoyjmX17KrWx2RrUjyZRtrf74vsyh4jPVhFRYMR4PSPLMUCnWD7V5XgZLZ1r3sHf8ydYPiqF1mGSJ4UvQRMtoWsg8uEhdzB85nyDW4BqoEWXPttbrmGx8wcwzmkuHAMFquB5FKgKvA34Vazk7Sv1f1yA3mL3ErPtHpadH67ciiwpBnsu6so5jFTxBoa6VFz5HTMLk9YGNYadoHok3PuBzJKZXkVKRWR38mD7CynHAxqDnYDwRLwdJpz1XVKU7i91aQgtEqgeef9cwsJfPr71vJ1o3YTL7xa7cWS9n163Tk29UHY9rsXQdExQAvSFB9G4UEHDFqfUStbzfdmkkMQtzjLfFKni25FhiotEaakuL1KTzd
```

Both payments are listed as included in the Mina explorer, however, the account overview is not showing any txs: https://minaexplorer.com/wallet/B62qoeGtw6SzaRahbprfy1RUgQBGHv7cRbzpPn5tLXdTxLSsL5R5q5o. Furthermore, the nonce on the account is now set to 2, even though no txs are shown.

**Comment by garethtdavies** [comment]
This is a bug on my explorer and not with Mina.

I actually detailed the root cause in this comment https://github.com/MinaProtocol/mina/pull/6754#issuecomment-730808802, which is the GraphQL `blocks` endpoint not returning the data. That endpoint has already been deprecated (as of last night :) ) 

The archive nodes run by O(1) Labs will have this data and so will count for any challenge purposes and the explorer is simply informational. That being said, I have a full history available and will clean up these few blocks (248->260) that I used that endpoint to import.
